recipe/meta.yaml

```diff
@@ -1,46 +1,43 @@
 {% set name = "GridDataFormats" %}
 {% set version = "0.4.1" %}
-{% set sha256 = "b362662c2dc475e2a3895fe044eaaa9a707bd660fd109a63dac84a47236690a3" %}
 
 package:
-  name: {{ name|lower }}
+  name: griddataformats
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/G/GridDataFormats/GridDataFormats-{{ version }}.tar.gz
+  sha256: b362662c2dc475e2a3895fe044eaaa9a707bd660fd109a63dac84a47236690a3
 
 build:
-  number: 0
+  number: 1
   noarch: python
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
-  build:
+  host:
```

Comment:
Python and pip should be in host, not build. See https://conda-forge.org/docs/maintainer/adding_pkgs.html#requirements for more info.

PS: the rest of my changes here are minor cosmetic changes.

Reply:
Many thanks, much appreciated.